static/js/upload_widget.js

```diff
@@ -61,8 +66,10 @@ var upload_widget = (function () {
                 input_error.hide();
             } else if (e.target.files.length === 1) {
                 var file = e.target.files[0];
-                if (file.size > 5 * 1024 * 1024) {
-                    input_error.text(i18n.t('File size must be < 5Mb.'));
+                if (file.size > max_file_upload_size * 1024 * 1024) {
+                     input_error.text(i18n.t('File size must be < __max_file_size__Mb.', {
+                        max_file_size: max_file_upload_size,
```

Comment:
There's various styles -- Zulip uses the trailing-comma-in-dicts style because it means that changes to add a new entry to the dict don't always end up patching an adjacent line.  In practice, this reduces merge conflicts and makes diffs more readable.